scripts/xtensa-build-all.sh

```diff
@@ -30,30 +39,55 @@ usage: xtensa-build.sh [options] platform(s)
        -u Force UP ARCH
        -d Enable debug build
        -c Interactive menuconfig
+       -o override defconfig with the config file in the src/arch/xtensa/configs/override/ directory
```

Comment:
Looking at the code I changed my mind sorry. How about something much more direct and less "magical"?
```
-o copies the file argument from src/arch/xtensa/configs/override/$arg.config
  to the build directory after invoking CMake and before Make.
```
Documentation normally abstracts implementation details away. However this script is just a convenience wrapper to save people calling CMake and then Make with the right parameters _and_ it is very common for many people to "fall down" to direct CMake and Make invocation - for instance every time they want to make... `.config` adjustments! So I think it's better to provide more transparency + show users how a very useful Kconfig trick works so people can re-use it manually when invoking CMake and Make directly. I mean I literally wish I had known about this override feature earlier.

I think it's also better to stay very far from any assumption about how this override config trick actually works: is it just a thin layer on top of the regular config or is the regular config completely ignored? Etc. All this belongs to CMakeLists.txt documentation, let's not go down this rabbit hole here.